### PR TITLE
Enhance fw info

### PR DESCRIPTION
**Pull request by JoeyZhang-Microsemi**
Add firmware map partition's information to fw-info and fw-update command.

### CODE REVIEW

**Comment by JoeyZhang-Microsemi** [comment]
Hi Logan,

Thanks a lot. I updated the code based on your comments.

Regards
Joey

File: inc/switchtec/switchtec.h [hunk 1]
```diff
@@ -53,6 +53,11 @@ struct switchtec_dev;
 #define SWITCHTEC_UNBOUND_PORT 255
 #define SWITCHTEC_PFF_PORT_VEP 100
 
+#define SWITCHTEC_BOOT_PART_START (0xa8000000)
+#define SWITCHTEC_MAP0_PART_START (0xa8020000)
+#define SWITCHTEC_MAP1_PART_START (0xa8040000)
+#define SWITCHTEC_PART_LEN (0x10000)
```
Comment (lsgunth) [inline inc/switchtec/switchtec.h]: Nit: you don't need parentheses around numbers in defines (only around expressions).

Comment (JoeyZhang-Microsemi) [reply inc/switchtec/switchtec.h]: Thanks.

**Comment by JoeyZhang-Microsemi** [comment]
Remove parentheses and update the name with _FLASH_

**Pull request merged**